python/py-hkdf/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-hkdf
+version             0.0.3
+platforms           darwin
+license             BSD
+maintainers         nomaintainer
+
+description         HMAC based Extract and Expand Key Derivation Function HKDF
+long_description    {*}${description}
+
+categories-append   security
+
+homepage            https://github.com/casebeer/python-hkdf
+master_sites        pypi:h/hkdf
+distname            hkdf-${version}
```

Comment:
Out of curiosity, is this still necessary? Seems to still fetch the file from `files.pythonhosted.org` without these lines.

Reply:
You're 100% right, @harens, I should have removed those.  Changes made.